library/src/scala/compiletime/ops/int.scala

```diff
@@ -181,4 +183,47 @@ object int:
    *  ```
    *  @syntax markdown
    */
+  @deprecated("Use compiletime.ops.any.ToString instead.","3.2.0")
```

Comment:
@soronpo I think this warning will be shown even before 3.2.0 is released which means users will have to choose between a deprecated and experimental, could you comment out the deprecated annotation for now?

Reply:
@smarter why would it show before? The version is specified. Is there a bug in `@deprecated`? If so, we should fix this instead.